client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -1,11 +1,13 @@
-import { isEnabled } from '@automattic/calypso-config';
+import config, { isEnabled } from '@automattic/calypso-config';
```

Comment:
No need for `config`

Reply:
AH I see, thank you! I misunderstood previously :)